docs/proposals/202105256-cluster-class-and-managed-topologies.md

```diff
@@ -129,13 +140,52 @@ For an end user, the UX to update the kubernetes version of the control plane an
 As an end user, I want to be able to easily scale up/down the number of replicas for each set of worker nodes in the cluster.
 - Currently, (for a cluster with 3 machine deployments) this is possible by updating these three different objects representing the sets of worker nodes in the pool. An easier user experience would be to update a single object to enable the scaling of multiple sets of worker nodes.
 
+#### Story 4 - Use ClusterClass to easily modify Clusters
+As an end user, I want to be able to easily change the configuration of all Clusters of a `ClusterClass`. For example, I want to be able to change the ControlPlane configuration by  
+changing the ControlPlane ref to point to another ControlPlane with a different configuration. The same should be possible for all templates referenced in a ClusterClass.
```

Comment:
>For example, I want to be able to change the ControlPlane configuration by changing the ControlPlane ref to point to another ControlPlane with a different configuration. The same should be possible for all templates referenced in a ClusterClass.

I feel this is describing a viable technical behaviour, can we try to elaborate why would one want to make use of this behaviour, what's the real world user story/scenario where this technical feature helps?

Reply:
You're right. I should have left the details how the config is changed out of it.